packages/app-root/src/hooks/usePanoptesUser.js

```diff
@@ -0,0 +1,74 @@
+import auth from 'panoptes-client/lib/auth'
+import { auth as authHelpers } from '@zooniverse/panoptes-js'
+import { useEffect, useState } from 'react'
+
+if (typeof window !== 'undefined') {
+  auth.checkCurrent()
+}
+
+async function fetchPanoptesUser() {
+  try {
+    const authorization = await auth.checkBearerToken()
+    /* 
+      Use crypto.subtle as a proxy for checking for SSL.
+      It will only exist for https:// URLs.
+    */
+    const isSecure = crypto?.subtle
+    if (authorization && isSecure) {
+      const { user, error } = await authHelpers.decodeJWT(authorization)
```

Comment:
https://github.com/zooniverse/panoptes/issues/4217

The user avatar is missing here.